spec/rubocop/cop/lint/duplicate_methods_spec.rb

```diff
@@ -0,0 +1,149 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Lint::DuplicateMethods do
+  subject(:cop) { described_class.new }
+
+  it 'registers an offense for duplicate method in class' do
```

Comment:
[two some_methods and two some_other_methods here](https://github.com/d4rk5eed/rubocop/blob/517f609377e1047bad42525fefd10ef192853d45/spec/rubocop/cop/lint/duplicate_methods_spec.rb#L169) and [messages compared here](https://github.com/d4rk5eed/rubocop/blob/517f609377e1047bad42525fefd10ef192853d45/spec/rubocop/cop/lint/duplicate_methods_spec.rb#L150), aren't they?